kitsune/questions/views.py

```diff
@@ -187,9 +187,15 @@ def question_list(request, product_slug):
             question_qs = question_qs.done()
 
     question_qs = question_qs.select_related("creator", "last_answer", "last_answer__creator")
-    question_qs = question_qs.prefetch_related("topic", "topic__product")
+    # Exclude questions over 90 days old without an answer or
+    # older than 5 years or
+    # created by deactivated users
+    today = date.today()
+    question_qs = question_qs.exclude(
+        created__lt=today - timedelta(days=90), num_answers=0, creator__is_active=False
```

Comment:
The `creator__is_active=False` exclude needs to be put in a separate `exclude()`, otherwise we're only excluding questions over 90 days old without an answer AND created by deactivated users.

And rather than doing `.exclude(creator__is_active=False)` I think it'd be preferable to avoid the double negative with `.filter(creator__is_active=True)`.